File: Web Alex Roel/s10-JS-POO/Encapsulamiento.js
class Persona{
    #nombre;//Atributo Privado 
    #edad;
    constructor(nombre , edad){
        this.#nombre = nombre;
        this.#edad = edad;
    }
    set setNombre(nombre){
        this.#nombre = nombre;
    }
    get getNombre(){
        return this.#nombre;
    }
    set setEdad(edad){
        this.#edad = edad;
    }
    get getEdad(){
        return this.#edad;
    }
}

const p1 = new Persona('Pablo', 21);
console.log(p1.getNombre);
p1.setNombre = 'Pepe';
console.log(p1.getNombre);

File: Web Alex Roel/s18-Python POO/encapsulamiento.py

class Persona:
    __nombre = None #Atributo privado
    __edad = None #Atributo privado

    def __init__(self, nombre, edad):
       self.__nombre = nombre
       self.__edad = edad
    
    #Metodo Privado
    def __metodo_privado(self):
        print('Soy un metodo privado')

    #MEtodos publicos

    def get_nombre(self):
        return self.__nombre
    
    def set_nombre(self, nombre):
        self.__nombre = nombre

    def get_edad(self):
        return self.__edad
    
    def set_edad(self, edad):
        self.__edad = edad


    def __str__(self):
        return f'Nombre: {self.__nombre} \nEdad: {self.__edad}'